scripts/docker-build-lib-linux.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -e
+
+IMAGE=${IMAGE:-manylinux2014_x86_64}
```

Comment:
`IMAGE` is essentially unused and just used to check if the host architecture (which could be obtained via `uname -m`) matches the target architecture. The build actually runs in a `centos:7` based container. 

It is a bit misleading that `manylinux2014` is being used here, but then the actual use is hardcoded below. The image name can for sure be hardcoded this in this script (it used to be originally), but we should get rid of the `IMAGE` env.

Reply:
This `$IMAGE` var is used to provide the name of the image which will be used to build the docker image for cross compilation. The default image is `centos:7`, but we use `manylinux2014_x86_64` since a lot of dependencies already installed inside which simplifies and speeds up preparation of the build environment.  

And also I would like us to be flexible enough with this , to be able to test different base images (e.g. locally) for the build without need to change anything.